crates/common/tedge_config/src/sudo.rs

```diff
@@ -54,6 +54,7 @@ impl SudoCommandBuilder {
         match which::which(self.sudo_program.as_ref()) {
             Ok(sudo) => {
                 let mut c = Command::new(sudo);
+                c.arg("--non-interactive");
```

Comment:
I'd recommend using the short form option `-n` here (even though it is less readable) because `-n` is more cross compatible, e.g. [freebsd](https://man.freebsd.org/cgi/man.cgi?query=sudo&apropos=0&sektion=0&manpath=FreeBSD+8.1-RELEASE+and+Ports&format=html) only has the `-n` option and not the long form.

Reply:
fixed